CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- [Add semantic tokens support from LSP](https://github.com/BetterThanTomorrow/calva/issues/1231)
+
```

Comment:
Did you read the Hack on Calva docs? 😄 The empty line between the header and the list breaks our CI. FYI. I'll fix this before releasing.